### PR TITLE
feat: same origing policy option

### PR DESCRIPTION
**Pull request by makovich**
Option to toggle same-origin policy. See [issue #42](https://github.com/igorlogius/igorlogius/issues/43).

### CODE REVIEW

**Comment by igorlogius** [comment]
Looks good!

I only saw a small typo with the spelling of "boundaries".

After that's fixed, i'll merge the PR.

File: options.html [hunk 1]
```diff
@@ -164,5 +164,17 @@
     <input type="number" id="multiopen3" value="4" min="0" max="99" />
   </fieldset>
 
+  <fieldset>
+    <legend>
+      <abbr title="Prevent data leakage on crossing web page origin bondaries">
```
Comment (igorlogius) [inline options.html:169]: small typo "boundaries"

Comment (makovich) [reply options.html]: oh! thanks. made `push --force` with fix

**Comment by makovich** [comment]
Fabulous! Updated the PR.

**Comment by igorlogius** [comment]
Github Actions will take care of the rest

**Pull request merged**